src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1575,6 +1635,11 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('server_key_type',
                    arg_type=server_key_type_param_type)
 
+        c.argument('auto_rotation_enabled',
+                   options_list=['--auto-rotation-enabled'],
```

Comment:
recommend `--enable-auto-rotation`

Reply:
i don't own this, we have multiple feature teams working on this. out of scope to make this change